docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
 add student n/<NAME> p/<PHONE_NUMBER> e/<EMAIL> sid/<STUDENT_ID>
 ```
 
-- Adding a student with the same student ID is not allowed.
-    - For example, if a student with a student ID of `A0123456X` already exists in CLIpboard, entering `add student n/Tom p/99887766 e/tom@example.com sid/A0123456X` will display an error message inside the result display.
+<div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
+
+  * Adding students with the same student id is not possible.
+  * The same student can exist in multiple groups. For example, a student can be in both tutorial group "T1" and a project group "Team15" at the same time.
+</div>
+<br>
 
 Examples:
+
 - `add student n/Tan Wei Xuan p/81776544 e/twx@example.com sid/A6788796F`
 - `add student n/Amirul Bin Othman p/98884417 e/amirul@example.com sid/A1324356G`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Did you know?:**
-The same student can exist in different groups, because that student can be in tutorial group T1 and project group Team15 at the same time.
-</div>
-<br>
-
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
 ### Deleting a student: `delete student`
 
-Deletes a student from the student roster.
+This removes a student from a selected group.
+
+**Command Format:**
 
-Format:
 ```
 delete student <INDEX>
 ```
 
 Examples:
-- `delete student 1` will delete the first student listed in the student list panel of CLIpboard.
 
-[Back to list of commands](#commands)
+- `delete student 1` will delete your first student listed in your student list.
+
+[Back to list of features](#features)
 
 ### Editing a student: `edit student`
 
-Did your student change their phone number, or you realised you misspelled their name? Do not worry, because you can easily change this with the edit command.
+If your student has changed their phone number, you may edit it, or any other student information with this command.
+
+**Command Format:**
 
-Format:
 ```
 edit student <INDEX> [n/<NAME>] [p/<PHONE_NUMBER>] [e/<EMAIL>] [sid/<STUDENT_NUMBER>]
 ```
+<div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
 
-- Edits student at index specified in `<INDEX>`.
-- At least one field `[n/<NAME>]`, `[p/<PHONE_NUMBER>]`, `[e/<EMAIL>]` or `[sid/<STUDENT_NUMBER>]` must be provided.
-- Fields entered following `edit <INDEX>` will replace the original fields.
-- Fields not entered will not replace the original fields.
+  * At least one parameter `[n/<NAME>]`, `[p/<PHONE_NUMBER>]`, `[e/<EMAIL>]` or `[sid/<STUDENT_NUMBER>]` must be provided.
+  * Parameters entered following `edit <INDEX>` will replace the original fields.
+  * Parameters not entered will not replace the original fields.
+</div>
+<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Made a typo? Try <code>undo</code> command!
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** Made a typo? Try the <code>undo</code> command!
```

Comment:
The `<br>` should come after `Tip` instead.

Currently it looks like this:
<img width="760" alt="image" src="https://user-images.githubusercontent.com/71481877/231633570-e5ea3ea2-e471-4c59-a90e-f98bdb195468.png">

What it would be better to look like:
<img width="761" alt="image" src="https://user-images.githubusercontent.com/71481877/231633647-d8e9628f-78dc-4728-af43-4207aefce4d2.png">

Reply:
fixed